lib/install/management/inspect.go

```diff
@@ -147,7 +147,7 @@ func (d *Dispatcher) GetDockerAPICommand(conf *config.VirtualContainerHostConfig
 			}
 
 			dEnv = append(dEnv, "DOCKER_TLS_VERIFY=1")
-			info, err := os.Stat(conf.ExecutorConfig.Name)
+			info, err := os.Stat(certpath)
```

Comment:
I feels like user does not have to specify cert-path if custom key/cert files are specified, so here I think the output should allow empty certpath case.